applications/app/services/TagFrontPicker.scala

```diff
@@ -1,15 +1,15 @@
 package services.dotcomrendering
 
 import common.GuLogging
-import experiments.{ActiveExperiments}
+import experiments.{ActiveExperiments, DCRTagFronts}
 import implicits.Requests._
 import play.api.mvc.RequestHeader
 import services.IndexPage
 
 object TagFrontPicker extends GuLogging {
 
   def getTier(tagFront: IndexPage)(implicit request: RequestHeader): RenderType = {
-    lazy val participatingInTest = false // There's no room for a 0% test at the moment - so we're just going with false
+    lazy val participatingInTest = ActiveExperiments.isParticipating(DCRTagFronts)
     lazy val dcrCouldRender = false
```

Comment:
Are we going to add any checks to get `dcrCouldRender` to true or set it to true? Otherwise [this condition](https://github.com/guardian/frontend/pull/26883/files#diff-070dfa195b4d051367265060055aa8e202997ddceba38be75c6b1a15f1699299R44) will always be false.

Reply:
Yes good spot, thank you, there was a separate ticket to find what needs to be excluded, I will raise a separate PR for that and mark this one as draft for now